todo/src/components/AddTodo.tsx

```diff
@@ -4,21 +4,24 @@ import { todoType } from "../App"
 interface IProps { todoList: todoType[], addTask: (taskName: string) => void }
 const AddTodo = ({ todoList, addTask }: IProps) => {
     const [str, setStr] = useState<string>('')
+    const[success,setSuccess] = useState<Boolean>(false)
     const btnClickHandler = () => {
         if (str !== "") {
             addTask(str)
             setStr('')
+            setSuccess(true)
+            setTimeout(()=>{setSuccess(false)},1000)
```

Comment:
why added timeout?

Reply:
so that success is set as false after the timeout and message that is been displayed "Todo added successfully" disappears